pipework

```diff
@@ -291,6 +287,12 @@ MTU=$(ip link show "$IFNAME" | awk '{print $5}')
 
 ip link set "$GUEST_IFNAME" netns "$NSPID"
 ip netns exec "$NSPID" ip link set "$GUEST_IFNAME" name "$CONTAINER_IFNAME"
+[ "$IFTYPE" = bridge ] && [ "$BRTYPE" = linux ] && [ "$VLAN" ] && {
+    ip netns exec "$NSPID" vconfig add "$CONTAINER_IFNAME" "$VLAN"
```

Comment:
Sweet! Just one thing: we should check if vconfig is available, and abort early if it's not available.

I would suggest to keep the IFTYPE/BRTYPE/VLAN check in the beginning, but instead of exiting, it would check if vconfig is available.